### DISCUSSION TITLE
Advanced Alerting - the Deployed State Connector - Stopped Alert - continues to Trigger and send email after connector has been moved to different channel

### DISCUSSION THREAD

**Discussion by JeffBenedict** [nextgenhealthcare/connect#5878] [Q&A]
Has anyone experienced this? If so, what is the solution to get the alerts to stop?
We are running MC 4.2 with the Advanced Alerting Extension. 
We were doing some channel/connector maintenance today, during that time our Deployed State - Stopped Alert triggered for a couple of connectors that were being moved.  After the maintenance was completed; the alerts still triggered for the connector that was moved.  Here's the overview. (names have been simplified )
Channel A has Destination Connectors A1, A2, A3
Channel B has Destination Connectors B1, B2

What we did was:
Exported Connector A3 from Channel A and Imported into Channel B
Disabled Connector A3 in Channel A
Deployed both updated Channels
After 10 min (threshold) the Deployed State Stopped Alert triggered on Channel A Connector A3, which made no sense, because we disabled the Connection.
So I removed the disabled Connector A3 from Channel A
Redeployed Channel A
So now our configuration is Channel A, Connector A1, A2
Channel B, Connector B1, B2, A3.
After 30 minutes, the alert fired again  on Channel A Connector A3, which is not correct because Connector A3 is now a destination on Channel B.

So my question is did I do something in the wrong order or something to confuse Advanced Alerting?  

I've tried the following and nothing has helped to stop the alerts
disabling the alert, saving, and re-enabling it and saving
disabling Channel A in the alert, saving, re-enabling, saving
changing the state to be checked from stopped to stopping , saving, changed it back and saved, 

I'm disabling this alert for the evening so I don't get emails all night.
Any help / guidance would be greatly appreciated.
Thanks.

**Comment by pacmano1** [comment]
That's a paid plugin, what did Nextgen support say?

**Comment by JeffBenedict** [comment]
I'm opening a case with them this morning. I'll keep this thread informed.
